functions/src/platform/assistant/handler.js

```diff
@@ -173,5 +172,9 @@ module.exports = (actionsMap) => {
     pipeline.stage(pipeline.IDLE);
   });
 
-  return functions.https.onRequest(bst.Logless.capture(functions.config().bespoken.key, app));
+  // dashbot.configHandler(app);
+
+  return functions.https.onRequest(bst.Logless.capture(functions.config().bespoken.key, app), (req, res) => {
```

Comment:
are you sure that `onRequest` takes 2 handlers? In docs there are only single handler https://firebase.google.com/docs/reference/functions/functions.https#.onRequest. if you are in doubt, you can you can take a look to source code here https://github.com/firebase/firebase-functions/blob/master/src/providers/https.ts#L36